mautic/mautic
# Fixed the regex to catch some email config messages that have % signs…

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | 
| Related user documentation PR URL | na
| Related developer documentation PR URL | na
| Issues addressed (#s or URLs) | na
| BC breaks? | na
| Deprecations? | na

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

When someone saves one of the email config settings for webview text, unsubscribe text, unsubscribe/resubscribe confirmation messages with two percent signs and spaces somewhere between, Symfony throws an exception that the parameter wasn't found.

#### Steps to test this PR:
1. Go to Configuration -> Email Settings -> and add the following to one of the message settings boxes `%hello there%`
2. Apply the configuration. It should have escaped with `%%hello there%%` which doesn't trigger the issue. 

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Repeat the above which will result in an exception
… in them which causes Symfony to choke
